ports/stm32/system_stm32.c

```diff
@@ -195,8 +197,11 @@ void SystemClock_Config(void) {
     __HAL_RCC_LSEDRIVE_CONFIG(RCC_LSEDRIVE_LOW);
     #endif
 
-    #if defined(STM32H7)
     // Wait for PWR_FLAG_VOSRDY
+    #if defined(STM32H7A3xx) || defined(STM32H7A3xxQ) || defined(STM32H7B3xx) || defined(STM32H7B3xxQ)
+    while (!__HAL_PWR_GET_FLAG(PWR_FLAG_VOSRDY)) {
```

Comment:
looks like this macro will work for all H7's, so could simplify this code block to:
```c
#if defined(STM32H7)
while (!__HAL_PWR_GET_FLAG(PWR_FLAG_VOSRDY)) {
}
#endif
```